cookbooks/aws-parallelcluster-scheduler-plugin/templates/default/compute_fleet_status/update-compute-fleet-status.erb

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash
+
+_error_exit() {
+  echo "ERROR: $1"
+  exit 1
+}
+
+_help() {
+    local -- _cmd=$(basename "$0")
+
+  cat << EOF
+
+  Usage: ${_cmd} [OPTION]...
+
+  Copy the AWS ParallelCluster Package to an S3 bucket.
+
+  --status                      Expected status of compute fleet
```

Comment:
this flag is to set status of the compute fleet. Please also add the accepted value, which is currently only PROTECTED